qa/tasks/cephfs/test_fscrypt.py

```diff
@@ -49,3 +49,29 @@ def test_fscrypt_encrypt(self):
         # skipped for now because of not supporting features in kernel or kceph.
         self.assertEqual(proc.returncode, 0)
         self.assertIn('Passed all 26 tests', stdout)
+
+    def test_fscrypt_dummy_encryption_with_quick_group(self):
+        self.require_kernel_mount()
+
+        self.write_local_config('test_dummy_encryption')
+
+        # XXX: check_status is set to False so that we can check for command's
+        # failure on our own (since this command doesn't set right error code
+        # and error message in some cases) and print custom log messages
+        # accordingly. This will take a long time and set the timeout to 5 hours.
+        proc = self.mount_a.client_remote.run(args=['sudo', './check',
+            '-g', 'quick', '-E', './ceph.exclude'], cwd=self.xfstests_repo_path,
+            stdout=StringIO(), stderr=StringIO(), timeout=18000, check_status=False,
```

Comment:
do we really need 5 hours timeout? doesn't it sound a lot?

Reply:
Sorry, it should be more than `2:50:46`. Please see https://pulpito.ceph.com/xiubli-2023-02-16_04:33:34-fs:fscrypt-wip-fscrypt-20230215-0834-distro-default-smithi/7175903/.

I will set it to 3 hours.